config/db.php

```diff
@@ -1,14 +1,14 @@
 <?php
 /**
- * @copyright Copyright (c) 2016 RNP
+ * @copyright Copyright (c) 2019 RNP
  * @license http://github.com/ufrgs-hyman/meican#license
  */
 
 return [
     'class' => 'yii\db\Connection',
-    'dsn' => 'mysql:host=localhost;dbname=meican31',
-    'username' => 'root',
-    'password' => 'root',
+    'dsn' => 'mysql:host=meican-db;dbname=your_database_name',
+    'username' => 'your_user',
+    'password' => 'your_password',
```

Comment:
I think you can just leave it like that. It's better than leaving it with the name 'root'

Reply:
Right. So I'll get the original code.